packages/docusaurus/src/server/i18n.ts

```diff
@@ -24,6 +24,8 @@ export function getDefaultLocaleConfig(locale: string): I18nLocaleConfig {
     label: getDefaultLocaleLabel(locale),
     direction: getLangDir(locale),
     htmlLang: locale,
+    // If the locale name includes -u-ca-xxx the calendar will be defined
```

Comment:
maybe add a test case for this? Using something like `ar-u-ca-islamicc` ?

+ test case for an invalid locale, ensuring it falls back and fail-safe instead of crashing

Reply:
We use `new Intl.DisplayNames` to construct the label, which is gonna crash on terribly malformed locale names anyways.

Reply:
This currently works fine: `yarn start:website --locale "My-weird-locale"`

But anything more fancy will crash: `yarn start:website --locale "My weird locale"`

But ok we can implement that later, considering we don't have anything setup to accept properly an invalid locale name